plugins/destination/postgresql/client/types.go

```diff
@@ -94,3 +94,86 @@ func (*Client) SchemaTypeToCockroach(t schema.ValueType) string {
 		panic("unknown type " + t.String())
 	}
 }
+
+func (c *Client) PgToSchemaType(t string) schema.ValueType {
+	switch c.pgType {
+	case pgTypeCockroachDB:
+		return c.CockroachToSchemaType(t)
+	default:
+		return c.Pg10ToSchemaType(t)
+	}
+}
+
+func (*Client) Pg10ToSchemaType(t string) schema.ValueType {
+	switch t {
+	case "boolean":
+		return schema.TypeBool
+	case "bigint", "integer":
+		return schema.TypeInt
+	case "double precision":
+		return schema.TypeFloat
+	case "uuid":
+		return schema.TypeUUID
+	case "text":
+		return schema.TypeString
+	case "bytea":
+		return schema.TypeByteArray
+	case "text[]":
+		return schema.TypeStringArray
+	case "timestamp without time zone":
```

Comment:
This will only become a problem with CDC, but at some point we'll also need to support (or at least handle without panicking?) all the other postgres types, like `timestamptz`.